From 837ae0e5beccc2f1e974afba56c22c3cf981fc4e Mon Sep 17 00:00:00 2001
From: Dmitry Baryshkov <dmitry.baryshkov@oss.qualcomm.com>
Date: Fri, 18 Sep 2026 17:52:15 +0300
Subject: [PATCH] qcom/shikra: link WiFi firmware from the ath10k subdir

On Qualcomm Shikra platform the WiFi firmware is loaded by the tqftpserv
from the same location as the rest of the DSP firmware. Add symlink,
pointing from the expected location to the actual WiFi firmware file.

Fixes: f64d7db7d264 ("ath10k: WCN3990: hw1.0: add shikra firmware files")
Signed-off-by: Dmitry Baryshkov <dmitry.baryshkov@oss.qualcomm.com>
Upstream-Status: Backport [https://gitlab.com/kernel-firmware/linux-firmware/-/commit/837ae0e5beccc2f1e974afba56c22c3cf981fc4e]
---
 WHENCE | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WHENCE b/WHENCE
index a712978b..053640a0 100644
--- a/WHENCE
+++ b/WHENCE
@@ -3825,6 +3825,7 @@ Version: WLAN.HL.2.0-01387-QCAHLSWMTPLZ-1
 File: ath10k/WCN3990/hw1.0/notice.txt_wlanmdsp
 File: ath10k/WCN3990/hw1.0/shikra/firmware-5.bin
 File: ath10k/WCN3990/hw1.0/shikra/wlanmdsp.mbn
+Link: qcom/shikra/wlanmdsp.mbn -> ../../ath10k/WCN3990/hw1.0/shikra/wlanmdsp.mbn
 Version: WLAN.HL.3.3.7.c5-00093.2-QCAHLSWMTPL-1
 File: ath10k/WCN3990/hw1.0/shikra/notice.txt_wlanmdsp
 
